roles/glance/templates/etc/glance/glance-api.conf

```diff
@@ -58,7 +58,7 @@ stores = glance.store.rbd.Store,
 rbd_store_ceph_conf = /etc/ceph/ceph.conf
 rbd_store_chunk_size = {{ glance.rbd_store_chunk_size }}
 rbd_store_user = glance
-rbd_store_pool = default
+rbd_store_pool = {{ ceph_default_pool }}
```

Comment:
Didn't see any failure with "store_smart" set to False and "store_ceph" set to True and running urusla with --tags=glance. glance-api.conf on controller nodes is updated with rbd store and ceph default pool,